modules/apps/frontend-editor/frontend-editor-alloyeditor-web/src/main/resources/META-INF/resources/alloyeditor.jsp

```diff
@@ -285,10 +285,10 @@ name = HtmlUtil.escapeJS(name);
 							show: 0
 						},
 						duration: 500,
-						message: '<liferay-ui:message key="your-request-failed-to-complete" />',
+						message: '<liferay-ui:message key="invalid-drag-and-drop-action" />',
```

Comment:
@jbalsas, @antonio-ortega, @jonmak08,

I didn't like the "failure feel/tone" of the previous message, as if the user was doing some valid action, and it just didn't succeed. I hope this new message better addresses what's expected of the user's action.